tools/ConfigureGeneratedApplications/ConfigurationJsonSerializerContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+using System.Text.Json;
+
+using ConfigureGeneratedApplications.Model;
+
+namespace ConfigureGeneratedApplications
+{
+    [JsonSourceGenerationOptions(ReadCommentHandling = JsonCommentHandling.Skip)]
+    [JsonSerializable(typeof(Configuration))]
+    [JsonSerializable(typeof(Project))]
+    [JsonSerializable(typeof(File))]
+    [JsonSerializable(typeof(PropertyMapping))]
+    [JsonSerializable(typeof(Replacement))]
+    [JsonSerializable(typeof(JsonElement))]
+    internal partial class ConfigurationJsonSerializerContext : JsonSerializerContext
+    {
```

Comment:
Should the option to ignore comments, which you deleted, be added here? Smth like https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/gladjohn-msi_v1//src/client/Microsoft.Identity.Client/Platforms/net/MsalJsonSerializerContext.cs#L48